easybuild/easyconfigs/m/MCU/MCU-3366c85-gomkl-2021a.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'PythonPackage'
+
+name = 'MCU'
+version = '3366c85'  # There is a 0.2 release but it's too old and it doesn't work
```

Comment:
@PetrKralCZ Don't use commit IDs as versions directly, since they don't sort.

Use a datestamp (date that the commit was created) as version instead, like this:

```python
local_commit = '3366c85'
version = '2021-04-06'
...
sources = [{'download_filename': '%s.tar.gz' % local_commit, 'filename': SOURCE_TAR_GZ}]